uportal-war/src/main/resources/logback.xml

```diff
@@ -146,11 +146,11 @@
   </logger>
   -->
   
-  <!-- Uncomment to see the XML at various stages in the rendering pipeline
+  <!-- Uncomment to see the XML at various stages in the rendering pipeline -->
```

Comment:
This is going to add a lot of noise to the default logs, isn't it?  Maybe better to leave this readily available in the comment but turned off by default?